docs/edgeql/index.rst

```diff
@@ -1,104 +1,85 @@
-
-So you can't create ``Movie`` first using DDL because it links
-to ``Person``.  The two are created in this order:
-
-.. code-block:: edgeql-repl
-
-    db> CREATE TYPE Person {
-    ...    CREATE REQUIRED PROPERTY name -> str;
-    ... };
-    CREATE: OK
-    db> CREATE TYPE Movie {
-    ...     CREATE REQUIRED PROPERTY title -> str;
-    ...     CREATE REQUIRED LINK director -> Person;
-    ...     CREATE MULTI LINK actors -> Person;
-    ... };
-    CREATE: OK
-
-SDL is sort of like a 3D printer: you set the final shape and it puts
-it together for you. DDL is like building a house with traditional
-methods: to add a window you first need a frame, to have a frame you
-need a wall, and so on. But DDL is great for making quick changes to
-your schema without a new migration, in the same way that you can
-replace a window without describing the whole house to do it.
-In practice, most people stick to SDL until they get comfortable
-and only then begin to experiment with DDL.
-
-**The remainder:** this includes infrequently used but critical items
-involved in database administration. Start with the
-**Administration** link below to see how to configure a database,
-set roles, passwords, and so on using EdgeDB.
-
-
 .. toctree::
     :maxdepth: 3
     :hidden:
 
-    concepts
-    expressions/index
-    commands/index
+    literals
+    sets
+    paths
     types
-    sdl/index
-    ddl/index
-    introspection/index
-    admin/index
-    lexical
-    output
-    eval
+    funcops
+    parameters
+    select
+    insert
+    update
+    delete
+    with
+    for
+    transactions
+
+EdgeQL is a next-generation query language designed to match SQL in power and
+surpass it in terms of clarity, brevity, and intuitiveness. It's used to query
+the database, insert/update/delete data, modify/introspect the schema, manage
+transactions, and more.
+
+Design goals
+------------
+
+EdgeQL is a spiritual successor to SQL designed with a few core principles in
+mind.
+
+**Compatible with modern languages**. A jaw-dropping amount of effort has been
+spent attempting to `bridge the gap <https://en.wikipedia.org/wiki/
+Object%E2%80%93relational_impedance_mismatch>`_ between the *relational*
+paradigm of SQL and the *object-oriented* nature of modern programming
+languages. EdgeDB sidesteps this problem by modeling data in an
+*object-relational* way.
+
+**Strongly typed**. EdgeQL is *inextricably tied* to EdgeDB's rigorous
+object-oriented type system. The type of all expressions is statically
+inferred by EdgeDB.
+
+**Easy to learn**. Postgres-flavored SQL contains `469 keywords
+<https://www.postgresql.org/docs/current/sql-keywords-appendix.html>`_. By
+comparison EdgeQL contains 80.
+
+**Designed for programmers**. It uses ``{ curly braces }`` to define scopes and
+nested structures and the *assignment operator* ``:=`` to set values. Plus it
+contains a comprehensive standard library of functions, operators, and control
+flow constructs.
```

Comment:
Well, SQL also contains a comprehensive standard library. Ours just way better structured and consistent. For example we always raise errors on invalid input (as opposed to some SQL functions returning NULL *sometimes*); we have a scheme for naming things and follow it; we have stricter typing (no implicit decimal <-> float casts).